### PR TITLE
Fix code-analysis

### PR DESCRIPTION
**Pull request by erral**
Fix code-analysis and sort imports

### CODE REVIEW

File: src/collective/privacy/browser/consent.py [hunk 1]
```diff
@@ -117,6 +114,6 @@ def getConsentRequired(self):
             try:
                 if not reason.isOpinionExpressed(self.request):
                     found.append(reason)
-            except:
+            except Exception:
```
Comment (hvelarde) [inline src/collective/privacy/browser/consent.py]: this is not a good, but I have no idea on what's the expected exception… can you leave a `# FIXME` comment?

File: src/collective/privacy/identifiers.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def getIdentifierForCurrentRequest(kls, request):
             if not email:
                 return None
             return kls.getIdentifierForUser(email)
-        except:
+        except Exception:
```
Comment (hvelarde) [inline src/collective/privacy/identifiers.py]: same here…

File: src/collective/privacy/storage.py [hunk 3]
```diff
@@ -1,9 +1,8 @@
 # -*- coding: utf-8 -*-
-
-import time
+from .tool import ProcessingReason
```
Comment (hvelarde) [inline src/collective/privacy/storage.py]: don't use relative imports.

File: src/collective/privacy/tests/test_robot.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
+from collective.privacy.testing import COLLECTIVE_PRIVACY_ACCEPTANCE_TESTING  # noqa
```
Comment (hvelarde) [inline src/collective/privacy/tests/test_robot.py]: this should be # noqa: E501

File: src/collective/privacy/tests/test_setup.py [hunk 5]
```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 """Setup tests for this package."""
+from collective.privacy.testing import COLLECTIVE_PRIVACY_INTEGRATION_TESTING  # noqa
```
Comment (hvelarde) [inline src/collective/privacy/tests/test_setup.py]: same here

File: src/collective/privacy/zcml.py [hunk 6]
```diff
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
+from collective.privacy.interfaces import ILawfulBasis, IProcessingReason
```
Comment (hvelarde) [inline src/collective/privacy/zcml.py]: please split this in 2 lines.

File: src/collective/privacy/browser/consent.py (hunk 1 above)
Comment (erral) [reply src/collective/privacy/browser/consent.py]: code-analysis was complaining about the bare exception but I didn't even know which exceptions were expected...

File: src/collective/privacy/zcml.py (hunk 6 above)
Comment (erral) [reply src/collective/privacy/zcml.py]: why code-analysis does not complain about this?

Comment (erral) [reply src/collective/privacy/zcml.py]: I have just seen that setup.cfg has some lines ignoring some violations.

Comment (MatthewWilkes) [reply src/collective/privacy/zcml.py]: I very much dislike how prescriptive some of the code analysis tools are by default, so on my projects I routinely disable some of the noisier checkers.

File: src/collective/privacy/browser/consent.py (hunk 1 above)
Comment (MatthewWilkes) [reply src/collective/privacy/browser/consent.py]: The only exception that is currently being raised is `ValueError`, which happens when trying to get the opinion or consent state of a user that isn't possible with the underlying configuration.

For example, if the item is stored in a cookie then it's only possible to find with `user=None`, if it's stored keyed on email then it's not always possible to determine with `user=None`. I'm not sure if `ValueError` is the right semantic choice here, the other potential is `KeyError` but I think that's a bit too far from the actual semantics of the call.

**Pull request merged**